php-cli.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1.3-labs
+FROM php:8.1-cli
+
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin/
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
+
+RUN <<EOF
+install-php-extensions \
+    apcu \
+    bcmath \
+    bz2 \
+    calendar \
+    ffi \
+    gd \
+    gmp \
+    imagick \
+    intl \
+    mysqli \
+    pcntl \
+    pcov \
+    pdo_mysql \
+    pdo_pgsql \
+    redis \
+    soap \
+    sockets \
+    sodium \
+    xsl \
+    zip \
+    exif
```

Comment:
I'm not sure, whether all of these are actually needed, so it might be possible to slim the image down and shorten the build duration a bit.